less/core/nav.less

```diff
@@ -5,9 +5,16 @@
     max-width: inherit;
   }
 
+  &__inner {
+    padding-left: @nav-btn-margin / 2;
+    padding-right: @nav-btn-margin / 2;
+  }
+
   &__btn {
+    margin: @nav-btn-margin (@nav-btn-margin / 2);
```

Comment:
did you mean `margin: calc(@nav-btn-margin / 2);` here?

Reply:
The `@nav-btn-margin` isn't a CSS variable which negates it needing the var prefix. 

The calc function also isn't required as the calculation is fairly basic (we're dividing an absolute number by an absolute number rather than say dividing a dynamic number (%) by an absolute number).

Reply:
@nav-btn-margin (@nav-btn-margin / 2); looked strange to me just, are we multiplying @nav-btn-margin by (@nav-btn-margin divided by 2)?

Reply:
if we are only dividing the CSS variable by 2 should it just be `margin: (@nav-btn-margin / 2)`?

Reply:
We're setting the vertical margin to match the value of `@nav-btn-margin` whilst the horizontal margin equals half of `@nav-btn-margin`.

Admittedly, it does look odd to have differing margins for the vertical and horizontal axis. I have added additional padding onto the `nav__inner` element to account for the reduced spacing to the left of the first element in the nav and to the right of the last as this was a cleaner method than trying to detect first and last visual elements.

Reply:
No problem, I realise now what this is doing! Apologies for initial the confusion.